src/gcp_scanner/crawl.py

```diff
@@ -1026,18 +1023,18 @@ def list_sourcerepo(project_id: str, credentials: Credentials) -> List[Any]:
   return list_of_repos
 
 
-def list_dns_policies(project_id: str, credentials: Credentials) -> List[Any]:
+def list_dns_policies(project_id: str,
+                      service: discovery.Resource) -> List[Any]:
   """Retrieve a list of cloud DNS policies in the project.
   Args:
     project_id: An id of a project to query info about.
-    credentials: An google.oauth2.credentials.Credentials object.
+    service: A resource object for interacting with the Compute API.
```

Comment:
This should say "DNS API", not "Compute API" right?

Reply:
updated